src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterFactory.cs

```diff
@@ -108,7 +112,15 @@ public virtual void Init(IProviderConfiguration providerCfg, string providerName
             adapterSettings = new EventHubStreamProviderSettings(providerName);
             adapterSettings.PopulateFromProviderConfig(providerConfig);
             hubSettings = adapterSettings.GetEventHubSettings(providerConfig, serviceProvider);
+#if NETSTANDARD
+            var connectionStringBuilder = new EventHubsConnectionStringBuilder(hubSettings.ConnectionString)
```

Comment:
They really changed. Everything that is on the old nugets that could be saved I did. They dont have those new patterns there even on the latest version of the old package...